### PR TITLE
Add user profile w.r.t the userID

### PR DESCRIPTION
**Pull request by Maheima**
The issue:
In the frontend we need data about user profile image and Name so, in the data that is returned from `/users/status` API, 
```js
{
  "currentStatus": {
     "message": String,
     "until": Timestamp,
     "updatedAt": Timestamp,
      "state": String,
      "from": Timestamp
    },
    "monthlyHours": {
       "updatedAt": Timestamp,
       "committed": Integer
    }
}
```

we will also return 
```js
{"currentStatus": {
                "message": String,
                "until": Timestamp,
                "updatedAt": Timestamp,
                "state": String,
                "from": Timestamp
            },
            "monthlyHours": {
                "updatedAt": Timestamp,
                "committed": Integer
            },
            "full_name": String,
            "picture": {
                "url": String,
                "publicId": String
            }
}
```

- Added picture field in users fixture because we were getting it in the Prod but there was no test for it.

### CODE REVIEW

**Review by RitikJaiswal75 — CHANGES_REQUESTED** [review]
Mahima please write tests supporting this.

File: test/fixtures/user/user.js [hunk 1]
```diff
@@ -29,6 +29,10 @@ module.exports = () => {
       },
       status: "active",
       profileURL: "https://abcde.com",
+      picture: {
+        publicId: "profile/mtS4DhUvNYsKqI7oCWVB/aenklfhtjldc5ytei3ar",
+        url: "https://res.cloudinary.com/realdevsquad/image/upload/v1667685133/profile/mtS4DhUvNYsKqI7oCWVB/aenklfhtjldc5ytei3ar.jpg",
```
Comment (Maheima) [inline test/fixtures/user/user.js:34]: Added picture field in users fixture because we were getting it in the Prod but there was no test for it.

**Review by heyrandhir — DISMISSED** [review]
LGTM

File: controllers/userStatus.js [hunk 2]
```diff
@@ -69,6 +70,12 @@ const getUserStatus = async (req, res) => {
 const getAllUserStatus = async (req, res) => {
   try {
     const { allUserStatus } = await userStatusModel.getAllUserStatus(req.query);
+    for (const doc of allUserStatus) {
```
Comment (rohan09-raj) [inline controllers/userStatus.js:73]: Can we name each doc as `user`?

File: test/integration/userStatus.test.js [hunk 3]
```diff
@@ -55,6 +55,10 @@ describe("UserStatus", function () {
           expect(res.body.totalUserStatus).to.be.a("number");
           expect(res.body.message).to.equal("All User Status found successfully.");
           expect(res.body.allUserStatus).to.be.a("array");
+          res.body.allUserStatus.forEach((doc) => {
```
Comment (RitikJaiswal75) [inline test/integration/userStatus.test.js:58]: let's use status instead of `doc`

**Review by RitikJaiswal75 — APPROVED** [review]
LGTM

**Pull request merged**